konsole/PKGBUILD

```diff
@@ -58,7 +59,8 @@ sha256sums=('fa0997c14a96252177e4808636ec43509eadb1482d54bb8a86b85810283f6cbc'
             'ee2217c8f4a49fc8c0f0f249daa601a3b05f6d7e2f0f7c26c0071bd975bf7358'
             'b6cb7b6d072396d44de0f097a27ae1546602730505c869f1d922c54d8df5f66a'
             '3f9f5f0c0d3df09432b5ccf80496c07069f9d78fbaf262d69ed6cfdc9a158fe6'
-            'af52aa81411395f7aec9621ff512d69d5cd9faad4ab9bf1f238f42fad8dd007d')
+            'af52aa81411395f7aec9621ff512d69d5cd9faad4ab9bf1f238f42fad8dd007d'
+            '0ef7aafaebbee42387a207fcacb711da61644f84285cfb160002679116996c7d')
```

Comment:
The hash here isn't valid. If you run `makepkg -g` It'll generate the hashes for you and you can just copy paste.